store/derivatives.ts

```diff
@@ -859,15 +862,32 @@ export const actions = actionTree(
       await this.app.$accessor.app.queue()
       await this.app.$accessor.wallet.validate()
 
-      await closePosition({
-        quantity,
-        price,
-        injectiveAddress,
-        address,
-        orderType,
-        market: (currentMarket || market) as UiDerivativeMarket,
-        subaccountId: subaccount.subaccountId
-      })
+      if (reduceOnlyOrders.length) {
```

Comment:
As stated above, make a new store action only for this case

Reply:
I made it happen